lib/core/storage/clientAdapter.ts

```diff
@@ -178,8 +230,8 @@ class ClientAdapter {
    *
    * @returns {Promise}
    */
-  async populateCache() {
-    const schema = await this.client.getSchema();
+  async populateCache(includeVirtual) {
```

Comment:
It would be better to have a named property here to give context when calling the function:
- `populateCache({ includeVirtual: true })` instead of `populateCache(true)`